plugins/widgetinspector/widget3dmodel.h

```diff
@@ -63,10 +63,7 @@ class Widget3DWidget : public QObject
     // QML does not handle 64bit integers, so use string instead, we only need
     // the value for comparison, we can convert back to quintptr in C++
     inline QString id() const {
-        QString str(8, QLatin1Char('0'));
-        const quint64 ptr = reinterpret_cast<quint64>(mQWidget.data());
-        std::memcpy(str.data(), &ptr, 8);
-        return str;
+        return QString::asprintf("%p", static_cast<const void*>(mQWidget.data()));
```

Comment:
This code isn't equivalent to the old version of the function. This converts the pointer into a user-readable string, while the old version (mis)used the string as a bytearray containing the pointer. It doesn't really matter, though, as it's only used for comparison as far as I see.